jupyter-book/glossary.md

```diff
@@ -104,20 +161,46 @@ Promoter
     Sequence of DNA to which proteins bind to initiate and control transcription.
 
 Pseudotime
-    Latent and therefore unobserved dimension reflecting cells' progression through transitions. Pseudotime is usually related to real time events, but not necessarily the same.
+    Latent and therefore unobserved dimension reflecting cells' progression through transitions.
+    Pseudotime is usually related to real time events, but not necessarily the same.
+
+RNA
+    Ribonucleic acid. Single-stranded nucleic acid present in all living cells that encodes and regulates gene expression.
```

Comment:
I think this is super fundamental and we can add 1-2 sentences of additional detail here.